pallets/emergency-shutdown/src/lib.rs

```diff
@@ -20,8 +20,9 @@
 
 //! Handle the ability to notify other pallets that they should stop all
 //! operations, or resume them
-
+#[cfg(feature = "runtime-benchmarks")]
```

Comment:
do you need that flag? It is normally showing up on top of the `benchmarking.rs` file

Reply:
Indeed not required, removed, Done :+1: